app/assets/scripts/views/locations-hub/location-card.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { PropTypes as T } from 'prop-types';
+import moment from 'moment';
+
+import { formatThousands } from '../../utils/format';
+import Card, { CardDetails, FooterActions } from '../../components/card';
+
+export default function LocationCard({
+  onDownloadClick,
+  lastUpdated,
+  name,
+  subtitle,
+  sourceType,
+  firstUpdated,
+  totalMeasurements,
+  parametersList,
+  id,
+  city,
+  countryData,
+  sourcesData,
+}) {
+  let updated = moment(lastUpdated).fromNow();
+  let started = moment(firstUpdated).format('YYYY/MM/DD');
+
+  const country = countryData || {};
+  let sources = [];
+  if (sourcesData.length) {
+    sourcesData.forEach((o, i) => {
```

Comment:
I think we can ask for this information to be included in the api response, rather than matching it across fetched data in the frontend (cc @bitner )

We would need per `/locations` location object:
- list with sources that contain url and name
- the country name (currently only has country code)

Reply:
is this for the locations/<id> endpoinst or the locations?location= endpoint?

Reply:
For the list of `/locations` used here https://openaq-staging.org/#/locations

Current response format 
```json
[{"id":2,"country":"GH","city":"Accra","cities":["Accra"],"location":"JTT - Jamestown","locations":["JTT - Jamestown"],"soureName":"Dr. Raphael E. Arku and Colleagues","sourceNames":["Dr. Raphael E. Arku and Colleagues"],"sourceType":"government","sourceTypes":["government"],"coordinates":{"longitude":-0.2129917,"latitude":5.5334556},"firstUpdated":"2006-11-13T07:31:00+00:00","lastUpdated":"2007-07-11T07:25:00+00:00","parameters":["pm25","pm10"],"countsByMeasurement":[{"parameter":"pm25","count":32},{"parameter":"pm10","count":32}],"count":64}, ... ]
```